### PR TITLE
Feat(assembly): xASTRO staked into vxASTRO is included in voting power

### PR DESCRIPTION
**Pull request by epanchee**

### CODE REVIEW

File: contracts/assembly/src/contract.rs [hunk 1]
```diff
@@ -773,14 +783,24 @@ pub fn calc_total_voting_power_at(deps: Deps, proposal: &Proposal) -> StdResult<
...
+
+        let locked_xastro: BalanceResponse = deps.querier.query_wasm_smart(
+            config.xastro_token_addr,
+            &XAstroTokenQueryMsg::BalanceAt {
+                address: vxastro_token_addr.to_string(),
+                block: proposal.start_block,
+            },
+        )?;
+
+        total = total.checked_add(locked_xastro.balance)?;
```
Comment (ShadoySV) [inline contracts/assembly/src/contract.rs:803]: We already added this at line 767

Comment (epanchee) [reply contracts/assembly/src/contract.rs:803]: They are different. Line 767 fetches xASTRO total supply while here we fetch xASTRO balance of vxASTRO contract

Comment (ShadoySV) [reply contracts/assembly/src/contract.rs:803]: But xASTRO balance of vxASTRO contract is already included in xASTRO total supply, isn't it?

Comment (epanchee) [reply contracts/assembly/src/contract.rs:803]: Hm, you are right. Thank you. Need to fix it

**Review by ShadoySV — APPROVED** [review]

**Review by maksk7 — APPROVED** [review]

**Pull request merged**